src/ec/packages/ec-component-breadcrumb/breadcrumb.story.js

```diff
@@ -0,0 +1,93 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import breadcrumbDocs from './docs/breadcrumb.md';
+import breadcrumb from './breadcrumb.html.twig';
+
+storiesOf('Components/Breadcrumb', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withNotes)
+  .addDecorator(withCode)
+  .add(
+    'simple',
+    () =>
+      breadcrumb({
+        links: [
```

Comment:
When possible, please use the data from the [spec component](https://github.com/ec-europa/europa-component-library/tree/v2/src/systems/ec/specs/components/breadcrumb/demo). That way, if we change data on ECL for whatever reason it will be updated everywhere.

You will probably have to map the structure of ECL data with yours, as it has been done [here](https://github.com/ec-europa/ecl-twig/blob/dee0125e5c2813591ba588b06797e8528af5eff8/src/ec/packages/ec-component-footer/demo/data.js#L5) or [here](https://github.com/ec-europa/ecl-twig/blob/bd05e41216fa121c5649c6f70ef5b42168aa6634/src/ec/packages/ec-component-social-media-share/demo/data.js#L4) for instance.

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/37/commits/cbbfd6766e07d2dbbb36be4083b7fcda656e7da3